package.json

```diff
@@ -128,8 +128,8 @@
   },
   "devDependencies": {
     "@ember/edition-utils": "^1.2.0",
-    "@octokit/rest": "^16.36.0",
-    "broccoli-plugin": "^3.0.0",
+    "@octokit/rest": "^17.1.0",
```

Comment:
Breaking changes listed here: https://github.com/octokit/rest.js/releases/tag/v17.0.0

Doesn't seem to affect us.